src/Input.test.jsx

```diff
@@ -0,0 +1,25 @@
+import { render, fireEvent } from '@testing-library/react';
+
+import Input from './Input';
+
+test('input', () => {
+  const taskTitle = '안녕';
+
+  const handleChange = jest.fn();
+  const handleClick = jest.fn();
+
+  const { container, getByPlaceholderText } = render(
+    <Input
+      value={taskTitle}
+      onChange={handleChange}
+      onClick={handleClick}
+    />,
+  );
```

Comment:
```suggestion
  const { container, getByPlaceholderText } = render((
    <Input
      value={taskTitle}
      onChange={handleChange}
      onClick={handleClick}
    />
  ));
```

괄호로 감싸면 콤마를 제거할 수 있어요!

Reply:
수정했습니다!
,를 줄이니 더 깔끔해진거같습니다!